main.go

```diff
@@ -176,11 +176,22 @@ func init() {
 		RunE:              contextualize(handler.Version, handler.Panic),
 	})
 
-	addRootCmd(&cobra.Command{
+	upCmd := &cobra.Command{
 		Use:   "up",
 		Short: "Upload and deploy project from the current directory",
 		RunE:  contextualize(handler.Up, handler.Panic),
-	})
+	}
+	upCmd.Flags().BoolP("detach", "d", false, "--detatch")
```

Comment:
I can't seem to find where this is used. Should it be removed?

Reply:
Removed. It's used in the tail up PR